modules/custom/cu_section_page_bundle/cu_section_page_bundle.info

```diff
@@ -0,0 +1,48 @@
+name = Section Page
+core = 7.x
+package = CU Section Page Bundle
+description = 
```

Comment:
Bundle needs a description. Currently `version = 7.x-1.1` is shown as the description